### ISSUE TITLE
Node.js workflow? [improvement]

### ISSUE THREAD

**Issue by AdvaitDhingra** [icecube/NuCat#11]
Since NuCat uses Node.js for both client and server side, it would make sense to have a workflow for it. 

Should someone (or me) add this?

**Comment by ChujieChen** [comment]
> Since NuCat uses Node.js for both client and server side, it would make sense to have a workflow for it.
> 
> Should someone (or me) add this?

I agree with you - it makes sense to have a workflow for the Node. We don't see any urgent need on our side but if you'd like to contribute, please feel free to add a YAML workflow for the Node. Thank you!
